Cargo.lock

```diff
@@ -504,7 +504,7 @@ checksum = "e78d4f1cc4ae33bbfc157ed5d5a5ef3bc29227303d595861deb238fcec4e9457"
 
 [[package]]
 name = "enveloper"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
:raised_eyebrow: I'd also question why this file is in the repo, as envelopers is a library crate.

Reply:
Very good point, I'll put up another PR to remove it